tuxemon/states/combat/combat.py

```diff
@@ -255,7 +255,7 @@ def __init__(
         self._new_tuxepedia: bool = False
         self._run: bool = False
         self._post_animation_task: Optional[Task] = None
-        self._xp_message = None
+        self._xp_message: Optional[str] = None
```

Comment:
Whoops, this attribute has been added by me... How did I forget the typehint.
Thanks for fixing it!